server/node/copy_from.go

```diff
@@ -74,9 +78,42 @@ func (cf *CopyFrom) Resolved() bool {
 	return true
 }
 
+// Validate returns an error if the CopyFrom node is invalid, for example if it contains columns that
+// are not in the table schema.
+//
+// TODO: This validation logic should be hooked into the analyzer so that it can be run in a consistent way.
```

Comment:
You can add this as an analyzer step in `server/analyzer`. Look at the `init.go` file to see how they're added.